### PR TITLE
Move to ghcr.io and use release drafter

### PR DESCRIPTION
**Pull request by mwindower**

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    - name: Lint
+      uses: golangci/golangci-lint-action@v2
+      with:
+        version: v1.32.2
+        args: --timeout=3m
+
+    - name: Build
+      run: |
+        export GITHUB_TAG_NAME=${GITHUB_REF##*/}
+        docker login -u ${{ secrets.DOCKER_HUB_USER }} -p ${{ secrets.DOCKER_HUB_TOKEN }}
```
Comment (Gerrit91) [inline .github/workflows/release.yaml:30]: Login should be removed from here.

**Review by Gerrit91 — CHANGES_REQUESTED** [review]

**Review by Gerrit91 — CHANGES_REQUESTED** [review]
The linter issue needs to be addressed.

File: go.mod [hunk 2]
```diff
@@ -13,12 +13,20 @@ require (
 	github.com/gobuffalo/envy v1.9.0 // indirect
 	github.com/gobuffalo/packr v1.30.1
 	github.com/gobuffalo/packr/v2 v2.8.0
-	github.com/golangci/golangci-lint v1.30.0 // indirect
+	github.com/google/go-cmp v0.5.0 // indirect
```
Comment (majst01) [inline go.mod:16]: Please run `go mod tidy`.

**Review by Gerrit91 — APPROVED** [review]

File: go.mod (hunk 2 above)
Comment (mwindower) [reply go.mod:16]: was already done

**Review by majst01 — APPROVED** [review]

**Pull request merged**